.ci/packer_cache.sh

```diff
@@ -13,6 +13,13 @@ cd "x-pack";
 yarn gulp prepare;
 cd -;
 
+# download chromedriver
+chromedriverVersion=2.42
```

Comment:
Why not just proceed with the download in `setup.sh` file? Or at least we can auto determine the chromedriver version to download, export env vars with all the info, and here we just need to create the dir and download accordingly those env vars. What do u think?

Reply:
The point of downloading it in the packer cache is to avoid downloading it when we run the jobs, spamming the download service, and sometimes getting back timeouts. If we put it in setup.sh then it will happen in every job.

Reply:
👍